lib/core/renderable/LocalPlayer.simba

```diff
@@ -1,13 +1,10 @@
 
 procedure TReflectLocalPlayer.Create;
 begin
-  if HookCache[TCacheKey.ckLocalPlayer][0] = 0 then
```

Comment:
The purpose of this method is to only create a new localplayer instance if it is needed. So if called during a script when the player is already created, nothing happens